### PR TITLE
Add new commandline arguments

### PR DESCRIPTION
**Pull request by tmke8**
* Specify model with commandline args

### CODE REVIEW

File: optimisation/loss.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import torchvision
 import torch.nn as nn
 from torch.nn import functional as F
-from utils.metrics import *
+from utils import metrics
```
Comment (Stoffle) [inline optimisation/loss.py]: Calling `getattr()` in a `models/<model>.py` won't be able to return functions from `metrics` with this change, is there a reason to keep them out of the `loss` namespace?

Comment (tmke8) [reply optimisation/loss.py]: Hm, I didn't think about that. I'll revert the change
